### PR TITLE
[MINI-4369] Switch Staging & Production Environment

### PR DESCRIPTION
**Pull request by laubengaier**
# Description
Implementing Switch for Staging and Production Environment

https://user-images.githubusercontent.com/17570451/142817400-501e41e5-eae9-403d-84be-45f75e138ca5.mp4

## Links
https://jira.rakuten-it.com/jira/browse/MINI-4360

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [ ] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by corycaywood — APPROVED** [review]
Looks good 👍

File: Example/Utils/Config.swift
Comment (Climbatize) [inline Example/Utils/Config.swift:24]: This might crash at first launch

```suggestion
        if let prod = UserDefaults.standard.bool(forKey: InternalKey.stagingActive.rawValue) {
            return prod
        } else {
            return false
        }
```

**Review by Climbatize — CHANGES_REQUESTED** [review]
LGTM except for the suggestion I provided. Also, before merging this one warn the team so we integrate the secrets in our environment ;)

File: Example/Utils/Config.swift
Comment (laubengaier) [reply Example/Utils/Config.swift:24]: @Climbatize 
Can you explain more in detail why it should crash? 
UserDefaults returns false for a non existing key so it behaves as expected using prod first. I've tested couple of times with deleting the app first.

Comment (Climbatize) [reply Example/Utils/Config.swift:24]: Ohhhhhh my bad !!! Friday review... I read a force unroll instead of a boolean manipulation 🤦‍♂️

**Review by Climbatize — APPROVED** [review]

**Pull request merged**
